src/main/java/frc/robot/subsystems/intake/IntakeOpenerSS.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import frc.robot.components.TrigonTalonSRX;
+import frc.robot.constants.RobotConstants.IntakeOpenerConstants;
+import frc.robot.subsystems.OverridableSubsystem;
+import frc.robot.utilities.EncoderConversions;
+
+public class IntakeOpenerSS extends OverridableSubsystem {
+    private final TrigonTalonSRX motor;
+
+    public IntakeOpenerSS() {
+        motor = IntakeOpenerConstants.MOTOR;
+    }
+
+    /**
+     * @param power to be applied to the motors
+     */
+    public void setPower(double power){
+        motor.set(power);
+    }
+    @Override
+    public void overriddenMove(double power) {
+        setPower(power);
+    }
+
+    /**
+     * @return the current angle of the intake
+     */
+    public double getAngle() {
+        // Converts position ticks (returned by the encoder) to an angle relative to the ground
+        // by timesing by 2048 (ticks in a circle) and dividing by 360 (degrees in  circle)
+        return EncoderConversions.MagToDegrees(motor.getSelectedSensorPosition());
+    }
```

Comment:
A. documentation is not required to a function that is written somewhere else that you did not change/modify
B. due to gear ratio 1 degree of the encoder does not equal 1 degree of the intake. either have it be in arbitrary ticks or do the conversion using a gear ratio

Reply:
a.but if we have documentation then if someone is reading the code then it will be easyer to understand it 
b. please explane the problem

Reply:
you didn't fix this....

Reply:
the gear ratio is 1,1